hack/build/cleanup_docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+REPO=
+
+function setTag {
+  if [ "$1" == "" ]; then
+      REPO="localhost"
+  else
+      REPO=$1
+  fi
+}
+
+function dockerCleanup {
```

Comment:
What does this do that "docker system prune -a" doesn't do?

Reply:
> What does this do that "docker system prune -a" doesn't do?

Prune will remove everything including k8s images, which can be reused in
opposite to cdi images that are built every cluster-sync and are set in
repo associated with generated local registry port of this specific cluster
instance

Reply:
this is a very cool feature.  But I think it should only prune images for the current (soon to be exiting) instance of the of the docker registry.  Can call gocli to get the registry port.  See here:  https://github.com/kubevirt/containerized-data-importer/blob/master/cluster/sync.sh#L16-L17

Reply:
> this is a very cool feature. But I think it should only prune images for the current (soon to be exiting) instance of the of the docker registry. Can call gocli to get the registry port. See here: https://github.com/kubevirt/containerized-data-importer/blob/master/cluster/sync.sh#L16-L17

Done. Please review